### ISSUE TITLE
Pie.Label requires react-native-skia font object in order to display data label text

### ISSUE THREAD

**Issue by coren-frankel** [FormidableLabs/victory-native-xl#368]
### My Environment

I've [forked the repository](https://github.com/coren-frankel/victory-native-xl/blob/ee0cfc930d17d57680ab1e6a1fb122e54db4da38/example/app/pie-chart.tsx#L58-L69) to demonstrate this in the example app with the Pie Chart page. I have not changed the behavior of the example but have added a benign conditional rendering to demonstrate my observation. The relevant snippet is in the code block below.

**What platform are you on?** (e.g., iOS, Android)
iOS and Android

### The Problem

In my attempts to use the brand new implementation of Pie.Label from #362 (Thank you @mwritter! 🙌), I struggled to implement the labels before eventually realizing that the font prop triggers the rendering of the pie slice labels. 

I now understand that voluntarily passing a react-native-skia font object to PieLabel is opting to have the label render, but that was not clear until I spent time looking at the code. I notice the font prop type is `undefined`, `null`, or `SkFont`, and I wonder if narrowing the type further would be better.

```tsx
...
export type PieLabelProps = {
  font?: SkFont | null;
  radiusOffset?: number;
  color?: Color;
  text?: string;
  children?: (position: LabelPostion) => ReactNode;
};
...
          <PolarChart
            data={data}
            colorKey={"color"}
            valueKey={"value"}
            labelKey={"label"}
          >
            <Pie.Chart>
              {({ slice }) => {
                return (
                  <>
                    <Pie.Slice>
                      {dataLabelSegment === "none" && (
                        // The pie label can be null or undefined to mute it,
                        // but in order to get the labels you must provide a font object
                        <Pie.Label
                          color={"black"}
                          // font={null}
                        />
                      )}
                      {dataLabelSegment === "simple" && (
                        <Pie.Label font={font} color={"black"} />
                      )}
                      {dataLabelSegment === "custom" && (
                        <Pie.Label radiusOffset={0.6}>
                          {(position) => (
                            <PieChartCustomLabel
                              position={position}
                              slice={slice}
                              font={font}
                            />
                          )}
                        </Pie.Label>
                      )}
                    </Pie.Slice>

                    <Pie.SliceAngularInset
                      angularInset={{
                        angularStrokeWidth: insetWidth,
                        angularStrokeColor: insetColor,
                      }}
                    />
                  </>
                );
              }}
            </Pie.Chart>
          </PolarChart>
```

I am currently building a react-native iOS/Android/Web app and have already opted for using the `victory` library for my web view and `victory-native` for mobile. My assumption that the new label component for this library was certainly based on the `victory/VictoryPie` component defaulting to render labels. Would it be prudent to share this default behavior and require a prop to be passed to  subdue labels? Or does this conflict with the design principles applied to native and mobile development?

I propose that either the docs be expanded on to explain this requirement or that the default behavior be altered, starting with the PieLabelProps font type.

**Expected behavior:** 
I expected to be able to skip passing a `font` prop to the `Pie.Label` component and still render the label from the data array passed to the `PolarChart` grandparent component.

**Actual behavior:** 
The Pie Label doesn't work without passing a SkFont object to each `Pie.Slice` in the render function.

### Additional Information

TL;DR: The new Pie.Label prop `font` is required to render labels on the Pie.Chart component but the documentation does not make this clear.

Thanks again for the fantastic library! 🙏

**Comment by mwritter** [comment]
@coren-frankel - Thanks for the feedback! 

`victory-native` uses Skia for rendering these graphs and labels, Skia `<Text />` elements require a `font` to render so a font will need to be provide somewhere, and `victory-native` doesn't offer any default theme or font, so the user will need to provide this.

Maybe the docs in `pie-chart.md` should be updated to make this more clear.

**Comment by coren-frankel** [comment]
> `victory-native` uses Skia for rendering these graphs and labels, Skia `<Text />` elements require a `font` to render so a font will need to be provide somewhere, and `victory-native` doesn't offer any default theme or font, so the user will need to provide this.

Thank you for breaking down what Skia is doing here; I had taken it for granted. And thanks for taking the time to consider a change.
